_posts/2019-11-27-update-sitemaps-during-heroku-deploy.markdown

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title:  "How to dynamically update your sitemap in Ruby"
+date: 2019-11-27 12:00:00
+categories: ["seo", "sitemap", "ruby"]
+author: "cleiviane"
+---
+
+A few months ago I received the task of making the [FastRuby.io](https://www.fastruby.io) sitemap refresh automatically after each deploy. That sounds like it would be pretty straightforward if we didn't have one issue (it's never that easy, right?). For the [FastRuby blog](https://www.fastruby.io/blog) we created a gem that encapsulates a [Jekyll](https://jekyllrb.com/) application. The discussion of why do we have a gem for our blog is actually a good topic for a new post. For now, I want to focus in the sitemap task that I had.
```

Comment:
```suggestion
A few months ago I received the task of making the [FastRuby.io](https://www.fastruby.io) sitemap refresh automatically after each deploy. That sounds like it would be pretty straightforward if we didn't have one issue (it's never that easy, right?). For the [FastRuby blog](https://www.fastruby.io/blog) we created a gem that encapsulates a [Jekyll](https://jekyllrb.com/) application. The discussion of why do we have a gem for our blog is actually a good topic for a new post. For now, I want to focus on the sitemap task that I had.
```